### PR TITLE
Exclude bare fed.us domain

### PR DESCRIPTION
**Pull request by jsf9k**
`fed.us` is really a first-level domain, analogous to `.gov` or `.com`.  It is only present in [`current-federal.csv`](https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv) as an accident of the way the registrar treats it.

Note that we are only removing the bare `fed.us` domain.  There are some proper TLDs such as `frb.fed.us` present in [`current-federal.csv`](https://github.com/GSA/data/blob/master/dotgov-domains/current-federal.csv) that remain.

This change has been tested by comparing the gatherer output to that from version 1.1.0.  The only difference is that `fed.us` is no longer present in `current-federal_modified.csv`.

### CODE REVIEW

File: gather-domains.sh [hunk 1]
```diff
@@ -42,6 +42,12 @@ cat $OUTPUT_DIR/current-federal.csv \
     /tmp/current-federal-non-dotgov.csv  > \
     $OUTPUT_DIR/current-federal_modified.csv
 ###
+# Remove the FED.US domain.  This is really a second-level domain,
```
Comment (dav3r) [inline gather-domains.sh]: I think you mean `This is really a top-level domain`

**Review by dav3r — APPROVED** [review]
Approved, though please see my comment correction 🇺🇸

File: gather-domains.sh (hunk 1 above)
Comment (jsf9k) [reply gather-domains.sh]: Yes, I will fix that now.

Comment (jsf9k) [reply gather-domains.sh]: Fixed in d74f81f.

**Pull request merged**